src/transactions/SetOptionsOpFrame.cpp

```diff
@@ -164,6 +166,17 @@ SetOptionsOpFrame::doApply(AbstractLedgerTxn& ltx)
         account.flags = account.flags | *mSetOptions.setFlags;
     }
 
+    // check clawback state after account.flags has changed
```

Comment:
comment should just be explicit on what the invariant is:
` ensure that revocable is set if clawback is set`

Reply:
Done